Toolbox/arcsdm/roctool.py

```diff
@@ -95,7 +95,7 @@ def execute(self, parameters, messages):
         pylab.legend(plot_handles, legend_items, loc=4)

```

Comment:
What if we fixed this line of code? Then there would not need to be a nested try-except block.

Reply:
(1) Calls to pylab have now been replaced by calls to matplotlib.pyplot (plt).
(2) Removed the try-except block which is should be redundant.